pkg/apis/eksctl.io/v1alpha5/validation_test.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 var _ = Describe("ClusterConfig validation", func() {
-	Describe("nodeGroups", func() {
+	Describe("nodeGroups[%d].name", func() {
```

Comment:
Minor nit: since this doesn't actually format it, should we use `[*]` to refer to the index?

```suggestion
	Describe("nodeGroups[*].name", func() {
```